src/utils/crypto.utils.ts

```diff
@@ -0,0 +1,10 @@
+import {uint8ArrayToHexString} from '@dfinity/utils';
+
+export async function generateHash(params: Record<string, string>): Promise<string> {
```

Comment:
- For consistency with the code base, can you declare it as arrow constant (`export const generateHash = async ()...` instead of "export function`?

- Should we implement it to be more generic? Like accepting any generic value that extends object?